backends/llamacpp/requirements.txt

```diff
@@ -1,3 +1,4 @@
 transformers==4.48.2
 huggingface-hub==0.28.1
 hf-transfer==0.1.9
+torch==2.6.0
```

Comment:
Naive Q: Do we really need `torch` package? It will certainly significantly increase the overall size of the container, so better asking 😅

Reply:
Yep required:
```
Traceback (most recent call last):
  File "/usr/bin/convert_hf_to_gguf.py", line 22, in <module>
    import torch
ModuleNotFoundError: No module named 'torch'
gguf_init_from_file: failed to open GGUF file 'models/model.gguf'
```